gh_pages/_source/session2/ros2/2-Launch-Files.md

```diff
@@ -89,28 +90,28 @@ You may also choose to launch `myworkcell_core` node with the others or keep it
 1. Now build the workspace to install the launch file and try to run it again:
 
    ``` bash
-   colcon build
    ros2 launch myworkcell_support workcell.launch.py
    ```
 
    _Note: Both nodes were automatically started. Press _Ctrl+C_ to close all nodes started by the launch file._
 
-1. Notice that none of the usual messages were printed to the console window.  Launch files will suppress console output below the **ERROR** severity level by default. To restore normal text output, add the `output='screen'` argument to each of the nodes in your launch files:
+1. The expected behavior is that you should see none of the usual messages printed to the console window.  Launch files will suppress console output below the **ERROR** severity level by default. To restore normal text output, add the `output='screen'` argument to each of the nodes in your launch file (see below).  _Current versions of ROS do not seem to follow this behavior, though, and output INFO/WARN messages to screen regardless of `output` setting._
```

Comment:
I believe this changed because debug/info used to come out on `stdout` and not on the screen by default, but now everything comes on `stderr`. I haven't looked into it by my guess is `output='screen'` is  the default now and you would only use that option to redirect everything to a file.